nipype/info.py

```diff
@@ -94,16 +95,16 @@ def get_nipype_gitversion():
 """
 
 # versions
-NIBABEL_MIN_VERSION = '2.0.1'
+NIBABEL_MIN_VERSION = '2.1.0'
 NETWORKX_MIN_VERSION = '1.7'
 NUMPY_MIN_VERSION = '1.6.2'
```

Comment:
We should probably raise to numpy>=1.11, I think the mmap issue was solved with that.